recipes/scippp/all/conanfile.py

```diff
@@ -0,0 +1,90 @@
+from conan import ConanFile
+from conan.errors import ConanInvalidConfiguration
+from conan.tools.build import check_min_cppstd
+from conan.tools.cmake import CMake, CMakeToolchain, cmake_layout
+from conan.tools.files import copy, get
+from conan.tools.microsoft import check_min_vs, is_msvc
+from conan.tools.scm import Version
+from os.path import join
+
+
+class ScipPlusPlus(ConanFile):
+    name = "scippp"
+    settings = "os", "compiler", "build_type", "arch"
+    generators = "CMakeDeps"
+    exports_sources = "CMakeLists.txt", "source/*", "include/*"
```

Comment:
`exports_sources` can be removed

Reply:
see https://github.com/conan-io/conan-center-index/pull/19458